hpcbench/cli/bensh.py

```diff
@@ -47,3 +47,4 @@ def main(argv=None):
         driver()
         if argv is not None:
             return driver
+        print(osp.abspath(driver.campaign_path))
```

Comment:
let's stick to loggers, no?

Reply:
`ben-wait` expects a campaign output-directory. `ben-sh` must provide it at some point. All messages emit by logging module are going to standard error (unless specified otherwise in CLI).
So IMHO printing the campaign path to standard output as it is will be very easy for the user to retrieve it programmatically.

Reply:
oh! I didn't understand what the print was doing here. In this case, that is fine. Please feel free to merge